src/components/onboarding/OnBoardingModal/OnBoardingModal.tsx

```diff
@@ -95,6 +102,9 @@ export default function OnBoardingModal({
   const [loading, setLoading] = useState(false)
   const [success, setSuccess] = useState(false)
 
+  const [loadingProxy, setLoadingProxy] = useState(false)
```

Comment:
if you can move the signer step continue button to Signer.tsx, then it should be possible also to move these states to Signer.tsx